datahub-web-react/src/app/ingest/source/builder/RecipeForm/SecretField/SecretField.tsx

```diff
@@ -34,32 +34,48 @@ interface SecretFieldProps {
     refetchSecrets: () => void;
 }
 
-function SecretField({ field, secrets, removeMargin, refetchSecrets }: SecretFieldProps) {
+function getTooltip() {
     return (
-        <>
-            <StyledFormItem name={field.name} label={field.label} tooltip={field.tooltip} removeMargin={!!removeMargin}>
-                <Tooltip title="This field requires you to use a DataHub Secret. For more information on Secrets in DataHub, please review the docs.">
-                    Secret Field
-                </Tooltip>
-                <Select
-                    showSearch
-                    filterOption={(input, option) => !!option?.children.toLowerCase().includes(input.toLowerCase())}
-                    dropdownRender={(menu) => (
-                        <>
-                            {menu}
-                            <StyledDivider />
-                            <CreateSecretButton refetchSecrets={refetchSecrets} />
-                        </>
-                    )}
+        <div>
+            <p>
+                This field requires you to use a DataHub Secret. For more information on Secrets in DataHub, please
+                review{' '}
+                <a
+                    href="https://datahubproject.io/docs/ui-ingestion/#creating-a-secret"
+                    target="_blank"
+                    rel="noreferrer"
                 >
-                    {secrets.map((secret) => (
-                        <Select.Option key={secret.urn} value={`\${${secret.name}}`}>
-                            {secret.name}
-                        </Select.Option>
-                    ))}
-                </Select>
-            </StyledFormItem>
-        </>
+                    the docs
+                </a>
+            </p>
+        </div>
+    );
+}
+
+function SecretField({ field, secrets, removeMargin, refetchSecrets }: SecretFieldProps) {
+    return (
+        <StyledFormItem name={field.name} label={field.label} tooltip={field.tooltip} removeMargin={!!removeMargin}>
+            <Tooltip title={getTooltip}>
+                <span style={{ color: ANTD_GRAY[7] }}>Secret Field</span>
```

Comment:
can we make this a styled component instead of inline?

Reply:
Fixed and Commited!